yellowbrick/regressor/residuals.py

```diff
@@ -479,42 +533,96 @@ def finalize(self, **kwargs):
         self.ax.set_ylabel('Residuals')
         self.ax.set_xlabel("Predicted Value")
 
-
-def residuals_plot(model, X, y=None, ax=None, **kwargs):
+        # Finalize the histogram axes
+        if self.hist:
+            self.hax.axhline(y=0, c=self.colors['line'])
+            self.hax.set_xlabel("Distribution")
+
+
+def residuals_plot(model,
+                   X,
+                   y,
+                   ax=None,
+                   hist=True,
+                   test_size=0.25,
+                   train_color='b',
+                   test_color='g',
+                   line_color=LINE_COLOR,
+                   random_state=None,
+                   **kwargs):
     """Quick method:
 
-    Plot  the residuals on the vertical axis and the
-    independent variable on the horizontal axis.
+    Divides the dataset X, y into a train and test split (the size of the
+    splits determined by test_size) then plots the training and test residuals
+    agains the predicted value for the given model.
 
     This helper function is a quick wrapper to utilize the ResidualsPlot
     ScoreVisualizer for one-off analysis.
 
     Parameters
     ----------
+    model : a Scikit-Learn regressor
+        Should be an instance of a regressor, otherwise a will raise a
```

Comment:
Line is not grammatically correct - "otherwise a will raise a" 
Same mistake also found at lines 62 & 330